### PR TITLE
Remove old code

### PR DESCRIPTION
**Pull request by HNOONa-0**
# Description

Removed code that is no longer necessary after successful migration.

Fixes: #7842

# Comments

Add any comments for the reviewers (e.g., indicating the beginning of the revision, something where the reviewer needs to pay special attention, etc.). If  there are no extra comments this section can be deleted.

# Checklist:

- [x] The code follows the [coding guidelines](https://github.com/omegaup/omegaup/wiki/Coding-guidelines) of omegaUp.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,7 +57,6 @@
     "bootstrap": "^4.6.0",
     "bootstrap-vue": "^2.21.2",
     "buffer": "^6.0.3",
-    "clean-webpack-plugin": "^4.0.0",
```
Comment (pabo99) [inline package.json:60]: Do you know what this library was used for and why it can now be removed?

**Review by pabo99 — APPROVED** [review]
We finally have a single configuration for the frontend with Webpack! 

Thank you for completing this entire project, Ahmed!

File: package.json (hunk 1 above)
Comment (HNOONa-0) [reply package.json:60]: Yes I installed that package when trying to hash the `ephemeral.js` script. But I did uninstall using yarn  , should I uninstall it using npm as well?

Comment (pabo99) [reply package.json:60]: No, that’s fine. Thanks for the explanation!

**Review by heduenas — APPROVED** [review]

**Review by heduenas — APPROVED** [review]

**Pull request merged**